src/confdb/confdb.c

```diff
@@ -2419,7 +2420,15 @@ static errno_t certmap_local_check(struct ldb_message *msg)
 
     tmp_str = ldb_msg_find_attr_as_string(msg, CONFDB_CERTMAP_MAPRULE, NULL);
     if (tmp_str != NULL) {
-        if (tmp_str[0] != '(' || tmp_str[strlen(tmp_str) - 1] != ')') {
+        /* The body of the rule must be enclosed with '(' and ')' but the rule
+         * may start with a prefix which ends with ':'. */
+        sep = strchr(tmp_str, ':');
+        if (sep != NULL && *sep != '\0') {
```

Comment:
The check for `*sep != '\0'` won't hurt but isn't really needed.

Reply:
Hi,

thanks for the comment, I must have mixed up `strchr()` with `strchrnull()`. The check is removed in the latest version.

bye,
Sumit